### PR TITLE
Improving results

### PR DESCRIPTION
**Pull request by rafaelsuchy**
The PR addresses the following:
- [x] Emax deviations plotted to scale
- [x] Convergence plots
- [x] Removal of outdated R_d sequence in notebooks
- [x] Update links in overview files
- [x] Update readme with proper links of nbviewer (maybe need to update after merge)
- [x] Split notebooks logically into data generation and data visualization of results (policy effect & convergence results)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[submodule "submodules/respy"]
+	path = submodules/respy
+	url = https://github.com/OpenSourceEconomics/respy.git
+        branch = quasi-monte-carlo
```
Comment (peisenha) [inline .gitmodules]: This is outdated, right?

File: python/auxiliary_figures.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    if "y" in which:
+        ax.get_yaxis().set_major_formatter(formatter)
+
+
+def get_fname(fname):
+    DIR_FIGURES = os.environ["DIR_FIGURES"]
+    return f"{DIR_FIGURES}/" + fname
+
+def get_single_integrand(df, idx, scaling, manual_limits="no", xlimits=[0,1], ylimits=[0,1]):
+    fig, ax = plt.subplots(1,1, figsize = (7, 4.5))
```
Comment (peisenha) [inline python/auxiliary_figures.py]: `figsize`handled once and for all in `matplotlib.rc` in configurations dir. Please remove manual specification everywhere.

File: python/auxiliary_figures.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+        ax.plot(x, y, label=label)
+
+    ax.legend()
+    if manual_limits == "yes":
+        ax.set_xlim(xlimits)
+        ax.set_ylim(ylimits)
+    ax.set_xlabel("Points")
+    ax.set_ylabel("Integrand")
+    #ax.set_ylim([380200, None])
```
Comment (peisenha) [inline python/auxiliary_figures.py]: Please remove all commented lines, check all files, please.

Comment (rafaelsuchy) [reply python/auxiliary_figures.py]: I removed commented lines that were outdated. I left commented lines that refer to commenting the code (happy to delete them too)

File: .gitmodules (hunk 1 above)
Comment (rafaelsuchy) [reply .gitmodules]: Specific branch not necessary anymore

**Pull request merged**